apps/heft/src/cli/HeftCommandLineUtilities.ts

```diff
@@ -249,60 +141,88 @@ export class HeftCommandLineUtilities {
   public registerStringListParameter(
     options: IRegisterParameterWithArgumentOptions
   ): IHeftStringListParameter {
-    const actionParameterMap: Map<CommandLineAction, CommandLineStringListParameter> = new Map();
-    for (const action of this._getActions(options.associatedActionNames)) {
+    return this._registerParameter(
+      options,
+      (action: CommandLineAction) => action.defineStringListParameter(options),
+      (parameter: CommandLineStringListParameter) => parameter.values,
+      (parameter: CommandLineStringListParameter) => (parameter.values || []).length > 0
+    );
+  }
+
+  private _registerParameter<
+    TTSCommandLineParameter extends CommandLineParameter,
```

Comment:
What does the `TS` in `TTS` stand for?

Reply:
TypeScript. The package is called `ts-command-line`.

Reply:
I see. I first thought `TS` probably stood for "TypeScript", but I wasn't sure why. I think `TCommandLineParameter` also works since it is extending from `CommandLineParameter`, but I don't have a strong opinion either way.

Reply:
I'll change it.